chromium_src/third_party/blink/renderer/modules/video_rvfc/video_frame_callback_requester_impl.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/modules/video_rvfc/video_frame_callback_requester_impl.h"
+
+#define kCoarseResolution kCoarseResolution [[maybe_unused]]
+
+#define FloorToMultiple(X) \
+  FloorToMultiple(         \
+      base::Microseconds(TimeClamper::CoarseResolutionMicroseconds()))
+
+#define static_assert(...) static_assert(true)
```

Comment:
this seems unsafe for several reason. First (from @cdesouza-chromium ) https://eel.is/c++draft/macro.names#2
and also this will potentially affect any static asserts in these headers (or any transitive includes)
```
#include "third_party/blink/renderer/modules/video_rvfc/video_frame_callback_requester_impl.h"

#include <memory>
#include <utility>

#include "base/trace_event/trace_event.h"
#include "third_party/blink/renderer/bindings/modules/v8/v8_video_frame_metadata.h"
#include "third_party/blink/renderer/core/dom/document.h"
#include "third_party/blink/renderer/core/dom/scripted_animation_controller.h"
#include "third_party/blink/renderer/core/execution_context/execution_context.h"
#include "third_party/blink/renderer/core/loader/document_loader.h"
#include "third_party/blink/renderer/core/timing/performance.h"
#include "third_party/blink/renderer/core/timing/time_clamper.h"
#include "third_party/blink/renderer/modules/video_rvfc/video_frame_request_callback_collection.h"
#include "third_party/blink/renderer/modules/xr/xr_frame_provider.h"
#include "third_party/blink/renderer/modules/xr/xr_session.h"
#include "third_party/blink/renderer/modules/xr/xr_system.h"
#include "third_party/blink/renderer/platform/wtf/functional.h"
```

Reply:
I think it's better to patch here. Redefining commonly used method names/keywords is always risky and while you could potentially avoid it by first including all the headers from the source file, that is very fragile.

Reply:
it is sometimes ok to do this if it gets overridden in a way that will obviously break for any unintentional overrides, but in this case it will just silently ignore them

Reply:
Good point. https://github.com/brave/brave-browser/issues/33358